recipe/meta.yaml

```
@@ -55,10 +55,12 @@ outputs:
         - pytorch {{ pytorch }} {{ torch_proc_type }}*
       run:
         - python
-        - pytorch {{ pytorch }} {{ torch_proc_type }}*
+        - pytorch
         - {{ pin_compatible('cudnn') }}          # [cuda_compiler_version != "None"]
         - {{ pin_compatible('numpy') }}
         - pillow >=5.3.0,!=8.3.0,!=8.3.1
+      run_constrained:
+        - pytorch * {{ torch_proc_type }}*
```

OTOH, should we add a lower bound here based on the version used to build? Not sure if `{{ pin_compatible() }}` would work out of the box, but it should, I guess?

---

as a rather irrelevant nit, I dislike significant whitespace. I tend to prefer
```suggestion
        - pytorch =*={{ torch_proc_type }}*
```

---

Done.